package com.esgi.nova.core.domain.users.exceptions

class UserNotFoundByUsernameAndPasswordException(username: String, password: String) : Throwable() {

}
